org.kartkrew.RingRacers.metainfo.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<component type="desktop-application">
+	<id>org.kartkrew.RingRacers</id>
+	<metadata_license>FSFAP</metadata_license>
+	<project_license>GPL-2.0+</project_license>
+	<name>Dr. Robotnik's Ring Racers</name>
+	<summary>Racing at the Next Level</summary>
+	<description>
+		<p>
+			Hang up your sneakers, because Dr. Robotnik and his ex-enemies
+			are going go-karting! Use the untapped potential of rings to
+			super-charge your vehicle across more than 200 crazy courses.
+			With over 20 unique items and some of the hottest moves on this
+			side of the Floating Island, this is <b>Racing at the Next Level!</b>
+		</p>
+		<p>
+			Ring Racers is packed to the brim with features, building off the
+			success of its predecessor SRB2Kart, including:
+		</p>
+		<ul>
+			<li>A ludicrously long list of courses and playable characters</li>
+			<li>A complete Grand Prix mode with CPU opponents and multiple difficulty levels</li>
+			<li>New and revamped items</li>
+			<li>Unique game systems and mechanics</li>
+			<li>Local split-screen multiplayer up to 4 players, and online play up to 16</li>
+			<li>A comprehensive Time Attack mode with replay support</li>
+			<li>A wealth of secrets to find</li>
+		</ul>
+		<p>
+			In the Flatpak version of Ring Racers, your game data is located in
+			<code>~/.var/app/org.kartkrew.RingRacers/.ringracers</code>. In order
+			to load user modifications, please store them in this directory, or
+			override your Flatpak configuration.
+		</p>
+		<p>
+			Kart Krew Dev is not affiliated with SEGA Corporation. Dr. Robotnik's Ring
+			Racers is a work of fan art.
+		</p>
+	</description>
+	<launchable type="desktop-id">org.kartkrew.RingRacers.desktop</launchable>
+	<screenshots>
+		<screenshot type="default">
+			<image>https://raw.githubusercontent.com/flathub/org.kartkrew.RingRacers/master/images/screenshot_1.png</image>
```

Comment:
this is not available as the repository doesn't exist.

screenshots should preferably be upstream too.

Reply:
this is part of why the build failed.